docs/source/examples/molecular_hydrogen.myst

```diff
@@ -85,31 +87,44 @@ radii = table[:, 0]
 coeffs = table[:, 1:]
 ```
 
-We define a function that evaluates the expectation value of the Hamiltonian on a noisy backend and for a given input circuit (ansatz).
-Here we use a Cirq density matrix simulator with a simple depolarizing noise model.
+We can also calculate their values automatically using OpenFermion (an open source quantum chemistry package).
+This results in slightly modified hamiltonians of the form:
+
+\begin{equation}
+    H = g_0 I + g_1 Z_0 + g_2 Z_1 + g_3 Z_0 Z_1 + g_4 X_0 X_1
+\end{equation}
 
 ```{code-cell} ipython3
-def energy(
-    ansatz: cirq.Circuit, radius_index: int, depo_noise_strength: float = 0.05
-) -> float:
-    """Computes the energy at a given bond length (radius)."""
-    npI = np.identity(2)
-    npX = np.array([[0.0, 1.0], [1.0, 0.0]])
-    npY = np.array([[0.0, -1j], [1j, 0.0]])
-    npZ = np.array([[1.0, 0.0], [0.0, -1.0]])
-    terms = [
-        coeffs[radius_index][0] * np.kron(npI, npI),
-        coeffs[radius_index][1] * np.kron(npZ, npI),
-        coeffs[radius_index][2] * np.kron(npI, npZ),
-        coeffs[radius_index][3] * np.kron(npZ, npZ),
-        coeffs[radius_index][4] * np.kron(npX, npX),
-        coeffs[radius_index][5] * np.kron(npY, npY),
-    ]
-    hamiltonian = sum(terms)
-    rho = compute_density_matrix(ansatz, cirq.depolarize, (depo_noise_strength,))
-    return np.trace(rho @ hamiltonian).real
+radii = [0.2 * i for i in range(1, 14)]
+
+def qubit_operator_to_pauli_sum(qubit_op) -> List:
+    """Converts the OpenFermion qubit operator to a list of mitiq Pauli Strings."""
+    psum = []
+    for ind_ops, coeff in qubit_op.terms.items():
+        if ind_ops == tuple():
+            psum.append(PauliString("I", coeff))
+            continue
+        term = ["I", "I"]
+        for ind, op in ind_ops:
+            term[ind] = op
+        term = ''.join(term)
+        psum.append(PauliString(term, coeff))
+    return psum
+
+def get_hamiltonian(bond_length) -> Observable:
+    """Returns the hamiltonian for the H2 molecule at the given bond length. Uses the STO-6G basis set and the Bravyi-Kitaev
+    transform to map fermionic operators to qubits. Results in a 2 qubit hamiltonian returned via a metriq.Observable."""
+    geometry = [('H', (0., 0., 0.)), ('H', (0., 0., bond_length))]
+    molecular_hamiltonian = generate_molecular_hamiltonian(geometry, basis='sto-6g', multiplicity=1, charge=0)
+    fermionic_op = of.transforms.get_fermion_operator(molecular_hamiltonian)
+    bk_op = of.transforms.symmetry_conserving_bravyi_kitaev(fermionic_op, active_orbitals=4, active_fermions=2)
+    qubit_operator = qubit_operator_to_pauli_sum(bk_op)
+    return Observable(*qubit_operator)
+
+hamiltonians = [get_hamiltonian(i) for i in radii]
```

Comment:
Is there any simple way to show that these Hemiltonians have indeed the structure mentioned in the latex formula?
E.g. something like 

`print(get_hamiltonian( some_both_lenght))`.

Or any other way to check which Hamiltonian OpenFermion is actually producing.

Reply:
I added a print function for the hamiltonian, so you can see it is of the same form. Let me know if that is what you were looking for.